miniwin/src/d3drm/backends/opengles3/renderer.cpp

```diff
@@ -610,6 +625,52 @@ void OpenGLES2Renderer::Resize(int width, int height, const ViewportTransform& v
 	}
 	m_renderedImage = SDL_CreateSurface(m_width, m_height, SDL_PIXELFORMAT_RGBA32);
 
+	if (m_colorTarget) {
+		glDeleteTextures(1, &m_colorTarget);
+	}
+	if (m_depthTarget) {
+		glDeleteRenderbuffers(1, &m_depthTarget);
+	}
+	if (m_msaaColorRbo) {
+		glDeleteRenderbuffers(1, &m_msaaColorRbo);
+	}
+	if (m_msaaDepthRbo) {
+		glDeleteRenderbuffers(1, &m_msaaDepthRbo);
+	}
+	m_colorTarget = m_depthTarget = m_msaaColorRbo = m_msaaDepthRbo = 0;
+
+	GLint samples, maxSamples;
+	glGetIntegerv(GL_SAMPLES, &samples);
+	glGetIntegerv(GL_MAX_SAMPLES, &maxSamples);
+	m_useMsaa = samples > 1;
+
+	if (m_useMsaa) {
+		glBindFramebuffer(GL_FRAMEBUFFER, m_msaaFbo);
```

Comment:
rather then having an FBO for msaa and one for non-msaa it would make more sens to have one FBO that is either MSAA or not, you won't ever use both at the same time so it's just taking up extra memory to have both.